apps/dhparam.c

```diff
@@ -7,14 +7,25 @@
  * https://www.openssl.org/source/license.html
  */
 
-/* We need to use some deprecated APIs */
+/* Sometimes we need to use some deprecated APIs */
```

Comment:
You're defining it down there for DSA. Is it needed also for other things than -dsaparam option? And if so, should it be TODO 3.0 to fix?

Reply:
It isn't needed at all.  In a no-deprecated build, no deprecated calls are made.  In a normal build it isn't required.